homeassistant/components/ai_speaker/__init__.py

```diff
@@ -0,0 +1,48 @@
+"""The AI Speaker integration."""
+import asyncio
+
+from aisapi.ws import AisWebService
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the AI Speaker integration."""
+    hass.data[DOMAIN] = {}
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up AI Speaker from a config entry."""
+    web_session = aiohttp_client.async_get_clientsession(hass)
+    ais = AisWebService(hass.loop, web_session, entry.data["host"])
```

Comment:
You should verify if you can connect or should raise ConfigEntryNotReady

Reply:
Also recommend that the loop shouldn't be passed around anymore. `asyncio.get_running_loop()` should be used instead nowadays.

Reply:
done